### DISCUSSION TITLE
Copy files and folders to local Files

### DISCUSSION THREAD

**Discussion by ikirja** [thebaselab/codeapp#560] [Q&A] [answered]
Is there a way to copy files and/or folders to local Files app on iPad? I haven’t found a way to do it. If there’s no such feature right now, is it on your roadmap?

**Comment by luni-moon** [comment]
> Is there a way to copy files and/or folders to local Files app on iPad? I haven’t found a way to do it. If there’s no such feature right now, is it on your roadmap?

Just use the actual files app to move it, the app has a folder in a default location of "On My iPad".

**Reply by ikirja** [reply]
I haven't managed to drag and drop from remote server to Files app. Nothing happens even if I hold long enough on folders/files that I need.![image](https://user-images.githubusercontent.com/44197850/188264718-b9183744-1018-4c71-9656-0db0e677038b.png)

**Reply by bummoblizard** [reply]
Hi. This feature isn't available yet. I will implement it in the next release.

**Reply by bummoblizard** [reply, marked as answer]
Implemented in the latest commit.
